### PR TITLE
Net 139 fsm dummy

### PR DESCRIPTION
**Pull request by LinnikD**
**- What I did**
v1 Net FSM implementation

### CODE REVIEW

File: network/nodenetwork/nodekeeper.go [hunk 1]
```diff
@@ -112,6 +112,10 @@ type nodekeeper struct {
 	syncLock sync.Mutex
 }
 
+func (nk *nodekeeper) IsBootstrapped() bool {
```
Comment (megge-dream) [inline network/nodenetwork/nodekeeper.go]: Add comment for this func

Comment (LinnikD) [reply network/nodenetwork/nodekeeper.go]: done

File: network/state/switcher.go [hunk 2]
```diff
@@ -17,19 +17,46 @@
...
+
+// OnPulse method checks current state and finds out reasons to update this state
+func (ns *NetworkSwitcher) OnPulse(ctx context.Context, pulse core.Pulse) error {
+	inslogger.FromContext(ctx).Info("Current NetworkSwitcher state is: %s", ns.state)
+
+	if ns.SwitcherWorkAround.IsBootstrapped() {
+		ns.stateLock.Lock()
+		ns.state = core.CompleteNetworkState
+		ns.stateLock.Unlock()
+		inslogger.FromContext(ctx).Info("Current NetworkSwitcher state switched to: %s", ns.state)
```
Comment (ivanshibitov) [inline network/state/switcher.go]: Concurrent access to ns.state
Save ns.state to another variable inside Lock and then use it here

Comment (ivanshibitov) [reply network/state/switcher.go]: Or even print it inside lock

**Pull request merged**